### PR TITLE
add cli

### PR DESCRIPTION
**Pull request by andrewharvey**

### CODE REVIEW

**Comment by andrewharvey** [comment]
@mollymerp this adds a cli, nothing relating to the library has changed so it shouldn't have any risk of breaking existing code

**Comment by mollymerp** [comment]
this looks good to me but admittedly I have never used this module – @tcql any objections to merging this?

File: tile-generator [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+const tileGenerator = require('./');
+const assert = require('assert');
+const argv = require('yargs')
+    .usage("$0 --minzoom 0 --maxzoom 10 --bbox east,south,west,north")
```
Comment (tcql) [inline tile-generator]: The ordering here is incorrect. Bounding boxes are specified in W, S, E, N order

File: tile-generator [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    .argv;
+
+const min_zoom = argv.minzoom;
+const max_zoom = argv.maxzoom;
+const bbox = argv.bbox.split(',').map((coord) => { return Number(coord); });
+
+assert(min_zoom <= max_zoom, 'min_zoom must be less than or equal to max_zoom');
+assert(min_zoom >= 0, 'min_zoom must be greater than or equal to 0');
+assert(bbox.length === 4, 'bbox must be 4 comma separated coordinates');
+assert(bbox[0] < bbox[2], 'bbox east must be less than bbox west');
```
Comment (tcql) [inline tile-generator]: The comparison here is correct (bbox[0] should be less than bbox[2]) but again the wording in incorrect: west is less than east

File: README.md [hunk 3]
```diff
@@ -29,3 +29,7 @@ tileGenerator(12)
   .on('data', function (tile) {
     console.log(tile);
   });
+
+### cli
```
Comment (tcql) [inline README.md]: looks like the code block in final example of the current README wasn't closed properly. My bad!

If you could, add a triple ``` ` ``` on the line before your `### cli` header, then also frame your example code with triple ``` ` ``` as well so it's formatted nicely

**Review by tcql — CHANGES_REQUESTED** [review]

**Comment by tcql** [comment]
@andrewharvey thank you for your contribution, by the way! Sorry it's taken so long to be reviewed (and thanks for the flag, @mollymerp)

File: tile-generator (hunk 1 above)
Comment (andrewharvey) [reply tile-generator]: I can't believe I stuffed that up, I mixed up my east and west!

File: tile-generator (hunk 2 above)
Comment (andrewharvey) [reply tile-generator]: fixed

File: README.md (hunk 3 above)
Comment (andrewharvey) [reply README.md]: fixed

**Comment by andrewharvey** [comment]
Thanks for reviewing @tcql , good job picking up on my errors.

**Review by tcql — APPROVED** [review]

**Comment by tcql** [comment]
Thanks! Merged & Published as v1.1.0

**Pull request merged**